src/lib.rs

```diff
@@ -251,3 +253,53 @@ impl LogWriter for CursiveLogWriter {
         log::LevelFilter::max()
     }
 }
+
+/// Show the flexi debug console.
+///
+/// This analog for `Cursive::show_debug_console`.
+pub fn show_flexi_debug_console(siv: &mut Cursive) {
+    siv.add_layer(
+        Dialog::around(FlexiLoggerView::scrollable().with_name(FLEXI_DEBUG_VIEW_NAME))
+            .title("Debug console"),
+    );
+}
+
+/// Show the flexi debug console, or hide it if it's already visible.
+///
+/// This analog for `Cursive::toggle_debug_console`.
+///
+/// # Enable toggleable flexi debug view
+///
+/// ```rust
+/// use cursive::Cursive;
+/// use cursive_flexi_logger_view::toggle_flexi_debug_console;
+/// use flexi_logger::{Logger, LogTarget};
+///
+/// fn main() {
+///     // we need to initialize cursive first, as the cursive-flexi-logger
+///     // needs a cursive callback sink to notify cursive about screen refreshs
+///     // when a new log message arrives
+///     let mut siv = Cursive::default();
+///
+///     Logger::with_env_or_str("trace")
+///         .log_target(LogTarget::FileAndWriter(
+///             cursive_flexi_logger_view::cursive_flexi_logger(&siv),
+///         ))
+///         .directory("logs")
+///         .suppress_timestamp()
+///         .format(flexi_logger::colored_with_thread)
+///         .start()
+///         .expect("failed to initialize logger!");
+///
+///     siv.add_global_callback('~', toggle_flexi_debug_console);  // Enable toggleable flexi debug view
+///
+///     // siv.run();
+/// }
+/// ```
+pub fn toggle_flexi_debug_console(siv: &mut Cursive) {
+    if let Some(pos) = siv.screen_mut().find_layer_from_name(FLEXI_DEBUG_VIEW_NAME) {
+        siv.screen_mut().remove_layer(pos);
+    } else {
+        show_flexi_debug_console(siv);
+    }
+}
```

Comment:
```rust
pub fn toggle_flexi_logger_debug_console(siv: &mut Cursive) {
    if let Some(pos) = siv.screen_mut().find_layer_from_name(FLEXI_LOGGER_DEBUG_VIEW_NAME) {
        siv.screen_mut().remove_layer(pos);
    } else {
        show_flexi_logger_debug_console(siv);
    }
}
```

Reply:
And we can extract here the method to simply remove the debug view from the current screen, so that we have something like this:
```rust
pub fn hide_flexi_logger_debug_console(siv: &mut Cursive) {
    if let Some(pos) = siv.screen_mut().find_layer_from_name(FLEXI_LOGGER_DEBUG_VIEW_NAME) {
        siv.screen_mut().remove_layer(pos);
    }
}
```

Reply:
I wrote exactly the same :)